tests/commented.feature

```diff
@@ -0,0 +1,76 @@
+# Feature: Testing Sphinx Writer
+
+#     Test the ability to write Gherkin data to a SphinxWriter object
+
+#     Background: Some requirements for this test
+#         Given that this file exists
+
+#     @quarantined @JIRA-1234
```

Comment:
This is OK, but I'm not sure if we need this many commented out scenarios... I don't object, but I don't think it's technically needed?

Reply:
I just copied the existing example - will update.